testing/util.py

```diff
@@ -41,6 +41,14 @@ def get_head_sha(dir):
         return cmd_output('git', 'rev-parse', 'HEAD')[1].strip()
 
 
+def normpath2(path):
+    """Like os.path.normpath, but does lower() for windows.  This is obsolete
+    in python3 which does this already.
+    """
+    path = os.path.normpath(path)
+    return path.lower() if os.name == 'nt' else path
```

Comment:
does `normcase` do the right thing?
https://docs.python.org/2/library/os.path.html#os.path.normcase

although...

> On Unix and Mac OS X, this returns the path unchanged; on case-insensitive filesystems, it converts the path to lowercase.

this seems a little dubious, doesn't OS X use a case-insensitive fs by default?